### PR TITLE
Fix sitemap generation

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra-front-bundle/pull/95
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/318
https://github.com/open-orchestra/open-orchestra-model-interface/pull/70

### CODE REVIEW

File: FrontBundle/Manager/SitemapManager.php [hunk 1]
```diff
@@ -72,46 +72,99 @@ public function generateSitemap(ReadSiteInterface $site)
     protected function getSitemapNodesFromSite(ReadSiteInterface $site)
     {
         $nodes = array();
-
-        // TODO : récupérer les noeuds en version published uniquement + vision publique
         $nodesCollection = $this->nodeRepository->findLastPublishedVersionByLanguageAndSiteId($site->getMainAlias()->getLanguage(), $site->getSiteId());
 
         if ($nodesCollection) {
             /** @var ReadNodeInterface $node */
             foreach($nodesCollection as $node) {
```
Comment (nicolasThal) [inline FrontBundle/Manager/SitemapManager.php]: Can you put a white space between foreach and `{`

**Comment by nicolasThal** [comment]
For the broken build, you can add a routine like the one set in the cms-bundle in the .travis.yml file

**Comment by ngilain** [comment]
@nicolasThal : update done

**Comment by nicolasThal** [comment]
nice :+1:

**Pull request merged**